src/main/modules/nunjucks/index.ts

```diff
@@ -19,11 +19,11 @@ export class Nunjucks {
       return typeof prop === 'function' ? prop(this.ctx) : prop;
     });
 
-    env.addGlobal('formItems', function (items: FormInput[]) {
+    env.addGlobal('formItems', function (items: FormInput[], selected: string) {
       return items.map(i => ({
         text: this.env.globals.getContent.call(this, i.label),
         value: i.value,
-        selected: i.selected,
+        checked: i.selected || i.value === selected,
```

Comment:
im confused on i.selected, would a form ever have a default value other than the one the user selected before and saved in session?

Reply:
i guess its to be generic component.

Reply:
Yeah, this is for generic input options. `i.selected` is from the `form` object in `content.ts`. So you could always make sure that an option is always selected by default. The second part is the users selection so I've renamed this to be `userSelection` in b07abed 👍